build/release/bootstrap-env.sh

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+
+# Store the current directory to reset to
+pushd $(pwd) > /dev/null
+
+# Change to the release directory
+cd "$(dirname ${BASH_SOURCE[0]})"
+
+# Import Scripts
+source "$(dirname '$0')/logging.sh"
+
+# Basic information
+SCRIPT_NAME="$(basename ${BASH_SOURCE[0]})"
+debugging "Script Name is ${SCRIPT_NAME}"
+
+###############################################################################
+# Function Declaration
+
+# set_build_variable - take 3 parameters 
+# the three parameters are 
```

Comment:
incomplete?

Reply:
I refactored and didn't update the signature. I'll fix